docs/Examples/Basic.md

```diff
@@ -163,7 +163,7 @@ var provider: MoyaProvider<MyService>!
 let provider = MoyaProvider(endpointClosure: endpointClosure)
 ```
 
-Neato. Now how do we make a request?
+Neat. Now how do we make a request?
```

Comment:
🤣 Neato is a valid slang though!

Reply:
Ohh I didn't know that 🤣 but apparently neither the MDSpell haha

Reply:
Reverted :)